addons/mail/static/src/views/web/form/form_renderer.js

```diff
@@ -1,7 +1,7 @@
 /* @odoo-module */
 
 import { AttachmentView } from "@mail/core/common/attachment_view";
-import { Chatter } from "@mail/core/web/chatter";
+import { Chatter } from "@mail/chatter/web_portal/chatter";
```

Comment:
There is a dependency issue here, it's not ok for a file in `core` (`views` is basically `core`, but badly split) to reference a file in a feature specific folder (`chatter`).

This form renderer file should probably be moved fully to `chatter/web` as well.